Bricknode.Soap.Sdk/Services/BfsOrderService.cs

```diff
@@ -368,6 +387,54 @@ public async Task<CreateAutoGiroDepositOrderResponse> CreateAutoGiroDepositOrder
             return response;
         }
 
+        /// <summary>
+        /// https://bricknode.atlassian.net/wiki/spaces/API/pages/3744595969/CancelAutoGiroOrders
+        /// </summary>
+        public async Task<CancelAutoGiroOrderResponse> CancelAutoGiroOrdersAsync(Guid[] orderIds, string? bfsApiClientName = null)
+        {
+            var request = await GetRequestAsync<CancelAutoGiroOrderRequest>(bfsApiClientName);
+
+            var listOfCancelTradeOrder = new List<CancelAutoGiroOrder>();
```

Comment:
This is autogiro but you named the variable "..TradeOrder"

Reply:
A copy&paste error, will fix it

Reply:
done